test/e2e/network/conntrack.go

```diff
@@ -275,6 +275,159 @@ var _ = SIGDescribe("Conntrack", func() {
 			framework.Failf("Failed to connect to backend 2")
 		}
 	})
+
+	// Regression test for #74839, where:
+	// Packets considered INVALID by conntrack are now dropped. In particular, this fixes
+	// a problem where spurious retransmits in a long-running TCP connection to a service
+	// IP could result in the connection being closed with the error "Connection reset by
+	// peer"
+	// xref: https://kubernetes.io/blog/2019/03/29/kube-proxy-subtleties-debugging-an-intermittent-connection-reset/
+	ginkgo.It("should drop invalid conntrack packets", func() {
+
+		// Create a TCP service
+		tcpJig := e2eservice.NewTestJig(cs, ns, serviceName)
+		ginkgo.By("creating a TCP service " + serviceName + " with type=NodePort in " + ns)
+		tcpService, err := tcpJig.CreateTCPService(func(svc *v1.Service) {
+			svc.Spec.Type = v1.ServiceTypeClusterIP
+			svc.Spec.Ports = []v1.ServicePort{
+				{Port: 80, Name: "tcp", Protocol: v1.ProtocolTCP, TargetPort: intstr.FromInt(80)},
+			}
+		})
+		framework.ExpectNoError(err)
+
+		// Store the script in a configmap so we can use it from the pods
```

Comment:
@claudiubelu PTAL, the script is a copy paste example I took from the internet, just focus on the approach, 
You just create a configmap and execute it from agnhost, it is more flexible than keep adding subcommands to agnhost, at least for this oneshot things that are not going to be reused.